packages/grafana-data/src/query/refId.ts

```diff
@@ -0,0 +1,20 @@
+import { DataQuery } from '@grafana/schema';
+
+export const getNextRefIdChar = (queries: DataQuery[]): string => {
```

Comment:
I think for exported funcs in a library we should add some kind of comment explaining what the function can be used for. That way it can come up in intellisense. I also wonder if this is the best name for this func. Would it be more accurate to call it like "createNewRefId" or "getNextRefId" since at first I thought maybe this was returning a char of an existing refid, and on closer look it seems like it can return more than one char.

Also maybe we can go to wherever the original is and right a note to say that this is now an exported function

Reply:
Great suggestions, made the changes!